vaxrank/epitope_prediction.py

```diff
@@ -109,6 +88,24 @@ def index_contains_kmer(fm, kmer):
         break
     return found
 
+def generate_protein_sequences(genome):
```

Comment:
thanks for fix! Could you put this into a separate PR that just deals with the shellinford issue?